docs/topics/secured-mission-intro-paragraph.adoc

```diff
@@ -1,6 +1,8 @@
-Mission proficiency level: 
+Mission proficiency level:
 //special case since topic is used by front end.
-ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
-ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-foundational}*].]
+ifdef::docs-topic[xref:proficiency_advanced[*{proficiency-advanced}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-advanced}*].]
 
 The Secured Booster expands on the {name-mission-http-api} Booster by securing a REST endpoint using link:https://access.redhat.com/products/red-hat-single-sign-on[{RHSSO}]. {RHSSO} implements the OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights to secured resources. Securing an application with SSO enables you to add security to your applications while centralizing the security configuration.
+
+IMPORTANT:  This  mission comes with {RHSSO} pre-configured for demonstration purposes. Accessing some of the components in this mission, for example, the web interface used for Authentication, requires changes to the default configuration.  Before using this mission, ensure that you are familiar with the basic concepts related to {RHSSO} configuration. 
```

Comment:
The provided configuration is suitable for all demonstration purposes, so "Accessing some of the components in this mission, for example, the web interface used for Authentication, requires changes to the default configuration" isn't exactly true. (Well there's a bug currently, but that's a bug, not an intention.)

I'd use something like this:

```adoc
IMPORTANT: This misson demonstrates using {RHSSO}, but doesn't explain its principles, usage or configuration.
Before using this mission, ensure that you are familiar with the concepts related to {RHSSO}.
```